package.json

```diff
@@ -11,9 +11,12 @@
   },
   "dependencies": {
     "@nuxt/ui": "^2.15.1",
+    "@vuelidate/core": "^2.0.3",
+    "@vuelidate/validators": "^2.0.4",
```

Comment:
please review these packages if ain't using them.

Reply:
removed it